### PR TITLE
Remove AFNetworking dependency for tests

### PR DESCRIPTION
**Pull request by chillpop**
In preparation for adding support for tvOS, I removed the dependency on AFNetworking.

While AFNetworking 3.x supports tvOS, it does not support iOS < 7.0.  Since this project supports iOS 6.0+, AFNetworking 2.* was installed as a dependency.

In addition, this PR updates the podspec and optionally uses a new method for percent encoding of URL Request parameters.

@vokal/ios-developers, code review? Relevant changes are in:
- podspec
- RequestBodyToFileNameTests.m
- VOKMockURLProtocol.m

I will put the additions needed to test on tvOS in another PR.

### CODE REVIEW

File: Example/Pods/Local Podspecs/VOKMockUrlProtocol.podspec.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "VOKMockUrlProtocol",
+  "version": "2.2.0",
+  "summary": "A url protocol that parses and returns fake responses with mock data.",
+  "homepage": "https://github.com/vokal/VOKMockUrlProtocol",
+  "license": {
+    "type": "MIT",
+    "file": "LICENSE"
+  },
+  "authors": {
+    "Vokal": "hello@vokalinteractive.com"
```
Comment (brockboland) [inline Example/Pods/Local Podspecs/VOKMockUrlProtocol.podspec.json]: May as well update this to hello@vokal.io while we're at it.

File: VOKMockUrlProtocol.podspec [hunk 2]
```diff
@@ -1,17 +1,18 @@
 Pod::Spec.new do |s|
   s.name             = "VOKMockUrlProtocol"
-  s.version          = "2.1.1"
+  s.version          = "2.2.0"
   s.summary          = "A url protocol that parses and returns fake responses with mock data."
   s.homepage         = "https://github.com/vokal/VOKMockUrlProtocol"
   s.license          = { :type => "MIT", :file => "LICENSE"}
   s.author           = { "Vokal" => "hello@vokalinteractive.com" }
```
Comment (brockboland) [inline VOKMockUrlProtocol.podspec]: Correction: hello@vokal.io update should happen here.

File: Example/Pods/Local Podspecs/VOKMockUrlProtocol.podspec.json (hunk 1 above)
Comment (brockboland) [inline Example/Pods/Local Podspecs/VOKMockUrlProtocol.podspec.json]: …but this ain't the place for that, now is it.

**Comment by brockboland** [comment]
LGTM aside from braaanding

**Comment by chillpop** [comment]
Branding updated

**Comment by brockboland** [comment]
LGTM. I think Travis was green but waiting just to make sure.

**Comment by brockboland** [comment]
Actually, one more quick change: could you switch it to ios@vokal.io instead?

**Comment by chillpop** [comment]
Done.  I also slipped in a change to silence a deprecation warning that already has a workaround.

**Comment by brockboland** [comment]
IS GOOD

**Pull request merged**